src/core/stdc/stdio.d

```diff
@@ -832,7 +910,8 @@ extern (D) @trusted
     ///
     int getchar()                 { return getc(stdin);     }
     ///
-    int putchar(int c)            { return putc(c,stdout);  }
+    int putchar(int c)            { return putc(c, stdout);  }
```

Comment:
I mentioned two spacing tweaks, this is the other one I don't see a need for.

Reply:
:-)